risc0/zkvm/Cargo.toml

```diff
@@ -23,6 +23,7 @@ protobuf-src = { version = "1.1", optional = true }
 anyhow = { version = "1.0", default-features = false }
 bytemuck = "1.12"
 cfg-if = "1.0"
+externc-libm = { git = "https://github.com/HaruxOS/externc-libm", rev = "873e2ee" }
```

Comment:
I think specifying a "git" package source dependency isn't allowed when publishing packages, so we should depend on this through crates.io like all our other dependencies.

Reply:
Hmmm that might be a blocker because: https://github.com/HaruxOS/externc-libm/issues/2#issuecomment-1480206267 causes a duplicate symbol error at link time with the existing crate.io published version.

Reply:
I'm not aware of crates.io preventing the use of git dependencies, but I also can't find any evidence for that in either direction.

Reply:
I think this is the tracking issue on `cargo` https://github.com/rust-lang/cargo/issues/6722